packages/devtools_app/lib/src/shared/device_dialog.dart

```diff
@@ -256,7 +255,7 @@ class _NameColumn extends ColumnData<_DialogFlag> {
         );
 
   @override
-  String getValue(_DialogFlag dataObject) => dataObject.name;
+  String? getValue(_DialogFlag dataObject) => dataObject.name;
```

Comment:
having these ColumnData overrides return null will likely cause issues with the table rendering. I'd set a default value (e.g. '')  instead and have these return types return a non-nullable string

Reply:
done

Reply:
This comment applies to all of the other getValue / getTooltip overrides too

Reply:
done